cpp/oneapi/dal/algo/svm/common.cpp

```diff
@@ -0,0 +1,126 @@
+/*******************************************************************************
+* Copyright 2020 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include "oneapi/dal/algo/svm/common.hpp"
+
+namespace oneapi::dal::svm {
+
+class detail::descriptor_impl : public base {
+public:
+    double c                         = 1.0;
+    double accuracy_threshold        = 0.001;
+    std::int64_t max_iteration_count = 100000;
+    double cache_size                = 200.0;
```

Comment:
don't we want to focus that it is in megabytes? For example, `cache_size_mb`. Asking because many entities in DPC++ and our library have `get_size()` method returning the size in bytes. I see possible confusion here.

Reply:
I think this should be reflected in the specification? I will think about your proposal, while in this PR I propose to leave it as it is.